plugins/modules/redfish_command.py

```diff
@@ -747,6 +777,8 @@ def main():
             new_username=dict(aliases=["account_username"]),
             new_password=dict(aliases=["account_password"], no_log=True),
             roleid=dict(aliases=["account_roleid"]),
+            account_types=dict(aliases=["account_accounttypes"]),
+            oem_account_types=dict(aliases=["account_oemaccounttypes"]),
```

Comment:
```suggestion
            account_types=dict(type='list', elements='str', aliases=["account_accounttypes"]),
            oem_account_types=dict(type='list', elements='str', aliases=["account_oemaccounttypes"]),
```

Reply:
I picked this up with my last commit to address CI errors.